ts/src/util.ts

```diff
@@ -135,29 +149,30 @@ export const dhtSync = async (
  */
 export const conductorCellsDhtSync = async (
   conductorCells: ConductorCell[],
-  interval = 50,
-  timeout?: number
+  intervalMs: number,
+  timeoutMs: number
 ) => {
   if (!isConductorCellDnaHashEqual(conductorCells)) {
     throw Error("Cannot compare DHT state of different DNAs");
   }
 
-  const startTime = performance.now();
-  let completed = false;
+  // Always run the check at least once, even if the timeoutMs is 0.
+  let completed = await areConductorCellsDhtsSynced(conductorCells);
 
+  const startTime = Date.now();
   while (!completed) {
     // Check if timeout has passed
-    const currentTime = performance.now();
-    if (timeout && Math.floor((currentTime - startTime) * 1000) >= timeout)
+    const currentTime = Date.now();
```

Comment:
Is there a specific reason for replacing `performance.now()` with `Date.now()` here? It probably shouldn't matter anyway...but I'm curious about the reason.

Reply:
I was just thinking we don't need microsecond precision, since the timeout is in millis. I agree probably doesn't really matter either way, was a tangent.